torchvision/prototype/transforms/_color.py

```diff
@@ -110,7 +110,7 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
             channel_permutation=torch.randperm(num_channels) if torch.rand(()) < self.p else None,
         )
 
-    def _permute_channels(self, inpt: features.ImageType, permutation: torch.Tensor) -> features.ImageType:
+    def _permute_channels(self, inpt: features.ImageOrVideoType, permutation: torch.Tensor) -> features.ImageType:
```

Comment:
We need this everywhere I think:
```suggestion
    def _permute_channels(self, inpt: features.ImageOrVideoType, permutation: torch.Tensor) -> features.ImageOrVideoType:
```